src/bundle/Resources/config/services/translation.yaml

```
@@ -9,6 +9,7 @@ services:
             - { name: jms_translation.file_visitor }
 
     Ibexa\AdminUi\Translation\Extractor\PolicyTranslationExtractor:
+        deprecated: 'Since ibexa/admin-ui 4.4: The "%service_id%" service is deprecated. Will be removed in 5.0'
```

For consistency with the rest of the files:
```suggestion
        deprecated: 'Since ibexa/admin-ui 4.4: The "%service_id%" service is deprecated, will be removed in 5.0'
```